### ISSUE TITLE
Adding Site Audit to kbox drush

### ISSUE THREAD

**Issue by possiBri** [kalabox/kalabox#1360]
Bug Report
==========

**Tell us about your setup**
What is your Kalabox version and operating system?

 _v0.12.0-beta1 on Mac 10.10.5_

Have you previously installed Kalabox? If so, what version?

_Yes, I've installed Kalabox v0.12.0-alpha8_

**Tell us about your bug**

_I've tried installing Site Audit to work with Drush so that I can get my site in better shape, but I can't seem to get Drush to recognize Site Audit._

**Please detail the steps we can take to replicate this bug. Be as detailed as possible.**
1. Downloaded Site Audit
2. Moved Site Audit (7.x) folder to my_app/config/drush folder
2b. Cleared Drush cache
2c. Run "aa" command didn't work, got error: _"Unhandled rejection VError: code: 1err:Couldn't connect to Docker daemon. You might need to install docker-osx"_
3. Searched/checked documentation for appropriate path setup
4. Tried putting Site Audit directory in my_app/config/drush/commands folder
4b. Cleared Drush cache
4c. Run "aa" command still doesn't work, but no cli error, generated html file just has error text: _Command aa needs the following extension(s) enabled to run: [31;40m[1m[error][0m site_audit. The drush command 'aa' could not be executed. [31;40m[1m[error][0m_
5. Tried using Terminus (following Pantheon tutorial), set up machine token with Pantheon, even after login is confirmed I get this same "[error] You are not logged in. Run `auth login` to authenticate or `help auth login` for more info" when I try to run  "sites aliases" command.
6. Tried updating Kalabox to beta1, but now Kalabox just hangs on opening =[[

**Tell us more**
Could better documentation have prevented this issue? if so what should we change?

_YES! Some documentation/clarity on how to add 3rd party commandfiles to the kbox drush would be super helpful, since the typical .drush folder was not created._

Thanks for all your hard work!

**Comment by pirog** [comment]
@possiBri as far as i know this SHOULD work and we have some tests to make sure you can drop custom command files in `config/drush`. I'll try to replicate this to see if there is additional mischief in play.

+1 for better documentation on this. docs are a weakness for us right now and we know that :(

for clarity after you drop the files in you are running `kbox drush aa`?

**Comment by possiBri** [comment]
Yes, both attempts were using the `kbox drush aa` command (plus some other params).

~~So does (based on your expectations that it should work) the custom command file need to be within a sub-directory called 'commands' or just the `config/drush` directory?~~

I was able to update to beta1 this morning after getting your assistance on a different issue when pulling Pantheon sites, so I went ahead and put Site Audit back in `config/drush` and ran `kbox drush aa` (plus html, file save info, etc), and this time I got a new error that wasn't related to Site Audit missing (some mPDF error). So maybe my old alpha install was just acting up? In fact, I just fixed the error before I finished writing this, and I'm looking at audit results... WOO!

Anyway, thanks for working so hard on all of this, and I appreciate that it's hard to get to documentation at the moment, but at least this can serve as temporary info for others!
